acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature

```diff
@@ -72,17 +74,30 @@ Feature: mod-consortia integration tests
     * call login universityUser1
     * call read('classpath:common-consortia/initData.feature@PutPermissions') { desiredPermissions: ['consortia.all']}
 
-  Scenario: Create consortium
+  @SetupConsortia
+  Scenario: Setup Consortia
+    # 1. Create Consortia
     * call read('tenant-utils/consortium.feature')
 
-  Scenario: Add 2 tenants to consortium
+    # 2. Add 2 tenants to consortium
     * call read('tenant-utils/tenant.feature')
 
-  Scenario: Add permissions to consortia_admin
+    # 3. Add permissions to consortia_admin
     * call read('tenant-utils/add-permissions-for-admin.feature')
 
+  Scenario: Prepare data
+    * call read('order-utils/inventory.feature')
+    * call read('order-utils/inventory-university.feature')
+    * call read('order-utils/configuration.feature')
+    * call read('order-utils/finances.feature')
+    * call read('order-utils/organizations.feature')
+    * call read('order-utils/orders.feature')
+
   Scenario: Create and open order
-    * call read('consortia-orders.feature')
+    Given call read('features/open-order-with-locations-from-different-tenants.feature')
+
+#  Scenario: Test cross-tenant inventory objects creation when working with pieces
+#    Given call read("features/pieces-api-test-for-cross-tenant-envs.feature")
```

Comment:
is this commented by mistake?

Reply:
thanks to point out. reverted back changes